### PR TITLE
chore: Use in the k8s manifest the ReadWriteOnce

### PR DESCRIPTION
**Pull request by krol3**
- Use ReadWriteOnce as a default for compatibility with more devices. https://kubernetes.io/docs/concepts/storage/persistent-volumes/#access-modes
- Improve the documentation. related the issue

### CODE REVIEW

File: docs/install.md [hunk 1]
```diff
@@ -31,6 +31,12 @@ kubectl create -f https://raw.githubusercontent.com/aquasecurity/postee/main/dep
 kubectl create -f https://raw.githubusercontent.com/aquasecurity/postee/main/deploy/kubernetes/hostPath/postee-pv.yaml
 ```
 
+>> Persistent volume
+>> - postee-db: persistent storage directory `/server/database`
+>> - postee-config: mount the cfg.yaml to a writable directory `/config/cfg.yaml`
+>> - postee-rego-templates: mount custom rego templates
+>> - postee-rego-filters: mount custom rego filters
+
```
Comment (simar7) [inline docs/install.md:39]:
```suggestion
!!! Note "Persistent Volumes Explained"
    - `postee-db`: persistent storage directory `/server/database`
    - `postee-config`: mount the cfg.yaml to a writable directory `/config/cfg.yaml`
    - `postee-rego-templates`: mount custom rego templates
    - `postee-rego-filters`: mount custom rego filters
```

**Comment by simar7** [comment]
Ping @krol3

**Comment by krol3** [comment]
> Ping @krol3

@simar7 I will review it on this week this PR

**Comment by simar7** [comment]
Could you try this on GCP too? Looks like @jessequinn did and shared this insight: https://github.com/aquasecurity/postee/issues/348#issuecomment-1146554091

**Comment by simar7** [comment]
> What is the reason we need to use ReadWriteMany? Sorry I'm not too familiar with it.

I guess to answer my question, would we need `ReadWriteMany` so that any future updates made to the config (via the UI) are updated back in the ConfigMap? If so should only the `postee-config` and `postee-db` volumes be `ReadWriteMany`?

File: deploy/kubernetes/postee.yaml [hunk 2]
```diff
@@ -192,9 +192,6 @@ spec:
               containerPort: 8445
               protocol: TCP
           volumeMounts:
-            - name: configmap-vol
-              mountPath: /configmap
-              readOnly: false
```
Comment (simar7) [inline deploy/kubernetes/postee.yaml:197]: Why did you remove this?

Comment (krol3) [reply deploy/kubernetes/postee.yaml:197]: In the initContainers/ setting-cfg we already copy the configmap, in Poste we are using the volume postee-config in the path /config
